modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java

```diff
@@ -849,10 +852,20 @@ public void testGetSearchContainerWithMultipleAuthors() throws Exception {
 					String.valueOf(_user.getUserId())
 				});
 
+			SearchContainer<Object> defaultSearchContainer =
+				_getSearchContainer(mockLiferayPortletRenderRequest);
+
+			JournalArticle journalArticle1 = JournalTestUtil.addArticle(
+				user.getUserId(), _group.getGroupId(), 0);
+			JournalArticle journalArticle2 = JournalTestUtil.addArticle(
+				_user.getUserId(), _group.getGroupId(), 0);
+
 			SearchContainer<Object> searchContainer = _getSearchContainer(
 				mockLiferayPortletRenderRequest);
 
-			Assert.assertEquals(2, searchContainer.getTotal());
+			Assert.assertEquals(
+				defaultSearchContainer.getTotal() + 2,
```

Comment:
Could we obtain defaultSearchContainer value, without calling the searchContainer, calling directly to a Journal method?